specification/redis/resource-manager/Microsoft.Cache/stable/2020-12-01/redis.json

```diff
@@ -1835,7 +1835,12 @@
         "replicasPerMaster": {
           "type": "integer",
           "format": "int32",
-          "description": "The number of replicas to be created per master."
+          "description": "The number of replicas to be created per primary."
+        },
+        "replicasPerPrimary": {
```

Comment:
This is a breakign change to an existing API, which is only allowed under special circumstances.

Reply:
@markcowl Are we talking about the changes in the description for the old property here?

Reply:
Intentended - merge target is branch Azure:dev-redis-Microsoft.Cache-2020-12-01